integration/proxy/proxy_helpers.go

```diff
@@ -593,12 +593,12 @@ func mustStartMockALBProxy(t *testing.T, proxyAddr string) *mockAWSALBProxy {
 }
 
 // waitForActivePeerProxyConnections waits for remote cluster to report a minimum number of active proxy peer connections
-func waitForActivePeerProxyConnections(t *testing.T, tunnel reversetunnel.Server, expectedCount int) {
+func waitForActivePeerProxyConnections(t *testing.T, tunnel reversetunnel.Server, expectedCount int) { //nolint:unused
```

Comment:
Why is the `unused` necessary here?

Could you add a comment after the directive explaining that? Eg: `//nolint:unused // <Explanation here>`. (See example in [Nolint Directive](https://golangci-lint.run/usage/false-positives/#nolint-directive).)